frontend/src/metabase-lib/v1/expressions/complete/functions.unit.spec.ts

```diff
@@ -88,13 +88,13 @@ describe("suggestFunctions", () => {
       {
         apply: expect.any(Function),
         detail:
-          'Returns the localized short name (eg. `"Apr"`) for the given month number (eg. `4`)',
-        displayLabel: "monthName",
+          "Looks at the values in each argument in order and returns the first non-null value for each row.",
```

Comment:
The order of the results changed slightly due the order of clauses changing. They are now order more alphabetically.